src/AboutContent.js

```diff
@@ -9,9 +9,36 @@ const AboutContent = function ( props ) {
     <wrapper>
       
       <Header as='h1' style={{ fontSize: '4em' }}>About the Cliff Effects Tool</Header>
-      <p style={{ fontSize: '1.33em' }}>
-        Fill in information about the tool here.
-      </p>
+
+      <Header as='h3'>What is this?</Header>
+      <p>This tool can help show how a change in income affects how much someone receives in public assistance from SNAP and Section 8 Housing Voucher benefits. It was designed for the case managers at <a href='http://www.prohope.org/'>Project Hope</a> with the aim of helping to predict changes in their client's benefits.</p>
+
+      <Header as='h3'>Why are you doing this?</Header>
```

Comment:
Bit of an odd phrasing. Sounds like [what the hero says](https://user-images.githubusercontent.com/2252820/33639202-30bda93e-d9f8-11e7-9575-19806ff33d87.png) in a plotline when they're backstabbed by a team member. I'd suggest the less pointed 'Why is this important?'.